### PR TITLE
Docs 1129 edits

### PR DESCRIPTION
**Pull request by csobier**
@rafatower , applied two minor updates as follows:
- changed default tool from Editor the Builder in the Overview section ([Docs issue#1129](https://github.com/CartoDB/docs/issues/1129))
- edited Postal Code Geocoder intro to be more generic ([Docs issue#1130](https://github.com/CartoDB/docs/issues/1130))

Let me know if you approve to merge? (Docs are generated from Master folder in this repo)

### CODE REVIEW

File: doc/geocoding_functions.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ INSERT INTO {tablename} (the_geom) SELECT cdb_geocode_namedplace_point('New York
 
 ## Postal Code Geocoder
 
-This function geocodes your data into point, or polygon, geometries for postal codes. The postal code polygon geocoder covers the United States, France, Australia and Canada; a request for a different country will return an empty response.
+This function geocodes your data into point, or polygon, geometries for postal codes. The postal code geocoder covers the United States, France, Australia and Canada; a request for a different country will return an empty response.
```
Comment (rafatower) [inline doc/geocoding_functions.md]: Old typo: "_These functions geocode_ ..." in plural

File: doc/overview.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ _The Data Services API is collaborating with [Mapzen](https://mapzen.com/), and
 
 By using the SQL API to query the Data Services API functions, you can manage specific operations and the corresponding geometries (a `polygon` or a `point`), according to the input information.
 
-The Data Services API decouples the geocoding and isoline services from the CARTO Editor. The API allows you to geocode data (from single rows, complete datasets, or simple inputs) and to perform trade areas analysis (computing isodistances or isochrones) programatically, through authenticated requests.
+The Data Services API decouples the geocoding and isoline services from the CARTO Builder. The API allows you to geocode data (from single rows, complete datasets, or simple inputs) and to perform trade areas analysis (computing isodistances or isochrones) programatically, through authenticated requests.
```
Comment (rafatower) [inline doc/overview.md]: I'd complete the paragraph with the other services: routing and Observatory (demographic and segmentation functions).

"decouples [...] from the CARTO Builder" is actually an architectural detail that goes beyond the scope of the DS API.

I'd rather say "The Data Services API exposes a number of services available to the Builder and to 3rd party applications...." or something along those lines.

Comment (csobier) [inline doc/overview.md]: If it's architectural, do we need to expose that in the public docs? If it's not account related, should we just delete this paragraph?

Comment (rafatower) [inline doc/overview.md]: > If it's architectural, do we need to expose that in the public docs?

we don't need to

> If it's not account related, should we just delete this paragraph?

It is not account related. To my understanding, it is up to you whether to rephrase it for accuracy and completeness or just remove it.

**Comment by csobier** [comment]
@rafatower , are these edits okay?

**Comment by rafatower** [comment]
yes, they are. Thanks a lot! :)

**Comment by csobier** [comment]
great, merging!

**Pull request merged**